packages/microcosm-www-next/src/pages/index.js

```diff
@@ -70,9 +77,7 @@ export default class IndexPage extends React.Component {
               dangerouslySetInnerHTML={{ __html: sectionData.heading }}
             />
 
-            <h3
-              className={'section__content__subheading -top' + browserClass}
-            >
+            <h3 className={'section__content__subheading -top' + browserClass}>
```

Comment:
Should there be a space here? Like will this turn into `-top<browserClass>`?

Reply:
nope because there is a space built into `browserClass` but I can see why that's confusing so I will change it

Reply:
Ah sorry :(. I must have missed it.